src/app/features/search/components/molecules/SearchResults/SearchResults.tsx

```diff
@@ -25,7 +28,12 @@ const mapData = (data: SearchResult) => [
 ]
 
 const SearchResults: React.FC<Props> = ({ searchString }) => {
-  const { data, isBusy } = useBAGWithZipCode(searchString)
+  const { data, isBusy, execGet } = useBAGWithZipCode(searchString)
+
```

Comment:
What does execGet do?

Reply:
It allows you to manually execute the endpoint's GET request. Good you asked, see I forgot to pass `lazy: true` to the hook options.